intro-HPC/ch_introduction.tex

```diff
@@ -153,20 +180,22 @@ \section{What is the \hpc?}
 \fi
 %TODO check whether others use accounting
 
-For maintenance and monitoring, we use:
 \ifantwerpen
+    For maintenance and monitoring, we use:
     \begin{enumerate}
       \item  Ganglia: monitoring software;
       \item  Icinga: alert manager.
     \end{enumerate}
 \fi
 \ifleuven
+    For maintenance and monitoring, we use:
     \begin{enumerate}
       \item  Ganglia: monitoring software;
       \item  Icinga: alert manager.
     \end{enumerate}
 \fi
 \ifbrussel
+    For maintenance and monitoring, we use:
```

Comment:
Can be done later, when we have something useful to add. e.g. user facing taccstats or ganglia